fabric-events-interaction-v0/src/client/java/net/fabricmc/fabric/mixin/event/interaction/client/ClientPlayerInteractionManagerMixin.java

```diff
@@ -64,33 +65,33 @@ public abstract class ClientPlayerInteractionManagerMixin {
 	private ClientPlayNetworkHandler networkHandler;
 	@Shadow
 	private GameMode gameMode;
+	@Shadow
+	private int blockBreakingCooldown;
```

Comment:
Where is this used?

Reply:
Oh nevermind, it should have been included in my revert commit 😓